doc/source/examples/multiphysics/air-bubble-compression/air-bubble-compression.rst

```diff
@@ -0,0 +1,337 @@
+          set Function expression = -0.0025
+        end
+      end
+    end
+
+Boundary Conditions - VOF
+************************************
+
+.. code-block:: text
+
+    subsection boundary conditions VOF
+      set number = 4
+      subsection bc 0
+        set id   = 0
+        set type = dirichlet
+        subsection dirichlet
+          set Function expression = 1
+        end
+      end
+      subsection bc 1
+        set id   = 1
+        set type = dirichlet
+        subsection dirichlet
+          set Function expression = 1
+        end
+      end
+      subsection bc 2
+        set id   = 2
+        set type = dirichlet
+        subsection dirichlet
+          set Function expression = 1
+        end
+      end
+      subsection bc 3
+        set id   = 3
+        set type = dirichlet
+        subsection dirichlet
+          set Function expression = 1
+        end
+      end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~
+
+In the ``physical properties`` subsection, we define the properties of the fluids. For air, represented by ``fluid 0``, the ``isothermal_ideal_gas`` density model is used to account for the fluid's compressibility.
+We refer the reader to the `Physical Properties - Density Models <https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html#density-models>`_ documentation for further explanation on the isothermal compressible density model.
+The properties of air and water at :math:`25 \, \text{°C}` are used in this example.
+
+.. code-block:: text
+
+    subsection physical properties
+      set number of fluids = 2
+      subsection fluid 0
+        set density model       = isothermal_ideal_gas
+        subsection isothermal_ideal_gas
+          set density_ref = 1.18
+          set R           = 287.05
+          set T           = 298.15
+        end
+        set kinematic viscosity = 0.0000156
+      end
+      subsection fluid 1
+        set density             = 1000
+        set kinematic viscosity = 0.000001
+      end
+    end
+
+Mesh
+~~~~
+
+In the ``mesh`` subsection, we define a hyper cube with appropriate dimensions. The mesh is initially refined :math:`8` times to ensure adequate definition of the interface.
+
+.. code-block:: text
+
+  subsection mesh
+    set type               = dealii
+    set grid type          = hyper_cube
+    set grid arguments     = -0.05 : 0.05 : true
+    set initial refinement = 8
+  end
+
+Mesh Adaptation
+~~~~~~~~~~~~~~~
+
+In the ``mesh adaptation`` subsection, adaptive mesh refinement is defined for the ``phase``. ``min refinement level`` and ``max refinement level`` are :math:`7` and :math:`9`, respectively. Since the size of the bubble changes, we choose a rather large ``fraction refinement`` (:math:`0.99`) and moderate ``fraction coarsening`` (:math:`0.01`).
+To capture the bubble adequately, we set ``initial refinement steps = 8`` to ensure that the initial condition is imposed for the VOF phase with maximal accuracy.
```

Comment:
Do you need the 8 refinement steps to have a good initial condition? Since you are starting at level 8 and your max refinement is 9, maybe less initial refinement steps can be used.

Reply:
I just checked, and it works well with lower values. I'll change it! :)